packages/salesforcedx-utils-vscode/src/helpers/utils.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+export function isEmpty(object: any): object is null | undefined {
```

Comment:
This name is a misnomer. We're not testing if an object is empty, but whether it is null or undefined. An empty object would be `{}`, and this function would return false for such a value. null and undefined are falsy values so it's fine to just test this using bang.
```
> !undefined
true
> !null
true
> !{}
false
```

Reply:
I think it's useful to keep this function since we have been using 'isNullOrUndefined' across the project. I'll adjust the name to be more representative of what the function is doing